setup.cfg

```diff
@@ -1,7 +1,3 @@
-[bdist_wheel]
-# we dont support py2
-universal = 0
```

Comment:
What its this bit used for? s it needed?

Reply:
was used for wheel build. now poetry is used

Reply:
So safe to remove?

Reply:
checked. yes, it's ok